layouts/_default/single.html

```diff
@@ -1,3 +1,54 @@
-{{ .Title }}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {{ partial "meta.html" . }}
```

Comment:
Should we add OG graph directly in this file with params in the .md files or we should add it in the meta.html? We need to add this somewhere, though, because when we will share on facebook it may not look how we want.